data-sources/mux/config.yml

```diff
@@ -0,0 +1,20 @@
+id: mux-video
+displayName: Mux
+description: |
+  Mux allows users to launch video features in days, customize the player experience, and monitor video streaming performance, all while scaling seamlessly to the world’s largest audiences.
+icon: logo.svg
+install:
+  primary:
+    link:
+      url: https://docs.newrelic.com/docs/logs/forward-logs/mux-video-streaming-firehose/
+keywords:
```

Comment:
Same with these keywords. A lot of them aren't on the list of automatically accepted keywords. 

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#keywords

@seemantk

Reply:
"quality of experience" is the only one that I'm not a huge fan of here, but the rest seem fine to me